examples/webgl_animation_scene.html

```diff
@@ -33,156 +33,124 @@
+			var renderer, mixer;
 
 			var clock = new THREE.Clock();
+			var url = 'models/json/scene-animation.json';
 
-			document.addEventListener( 'mousemove', onDocumentMouseMove, false );
+			var SCREEN_WIDTH = window.innerWidth;
+			var SCREEN_HEIGHT = window.innerHeight;
 
-			init();
-			animate();
+			var container = document.getElementById( 'container' );
 
-			function init() {
 
-				container = document.getElementById( 'container' );
+			stats = new Stats();
+			container.appendChild( stats.dom );
 
-				camera = new THREE.PerspectiveCamera( 30, SCREEN_WIDTH / SCREEN_HEIGHT, 1, 10000 );
-				camera.position.z = 150;
+			renderer = new THREE.WebGLRenderer( { antialias: true } );
+			renderer.setClearColor( 0xffffff );
+			renderer.setPixelRatio( window.devicePixelRatio );
+			renderer.setSize( SCREEN_WIDTH, SCREEN_HEIGHT );
+			container.appendChild( renderer.domElement );
 
-				scene = new THREE.Scene();
 
-				scene.fog = new THREE.Fog( 0xffffff, 2000, 10000 );
+			// Load a scene with objects, lights and camera from a JSON file
 
-				//scene.add( camera );
+			new THREE.ObjectLoader().load( url, function ( loadedScene ) {
 
-				// GROUND
+				scene = loadedScene;
 
-				var geometry = new THREE.PlaneBufferGeometry( 16000, 16000 );
-				var material = new THREE.MeshPhongMaterial( { emissive: 0x000000 } );
+				// If the loaded file contains a perspective camera, use it with adjusted aspect ratio...
 
-				var ground = new THREE.Mesh( geometry, material );
-				ground.position.set( 0, FLOOR, 0 );
-				ground.rotation.x = -Math.PI/2;
-				/*scene.add( ground );*/
+				scene.traverse( function ( sceneChild ) {
 
-				ground.receiveShadow = true;
+					if ( sceneChild.type === 'PerspectiveCamera' ) {
 
+						camera = sceneChild;
+						camera.aspect = SCREEN_WIDTH / SCREEN_HEIGHT;
+						camera.updateProjectionMatrix();
 
-				// RENDERER
+					}
 
-				renderer = new THREE.WebGLRenderer( { antialias: true } );
-				renderer.setClearColor( scene.fog.color );
-				renderer.setPixelRatio( window.devicePixelRatio );
-				renderer.setSize( SCREEN_WIDTH, SCREEN_HEIGHT );
-				renderer.domElement.style.position = "relative";
+				} );
 
-				container.appendChild( renderer.domElement );
+				// ... else create a new camera and use it in the loaded scene
 
-				renderer.gammaInput = true;
-				renderer.gammaOutput = true;
+				if ( camera === undefined ) {
 
-				renderer.shadowMap.enabled = true;
+					camera = new THREE.PerspectiveCamera( 30, SCREEN_WIDTH / SCREEN_HEIGHT, 1, 10000 );
+					camera.position.set( - 200, 0, 200 );
 
+				}
 
-				// STATS
+				controls = new THREE.OrbitControls( camera );
 
-				stats = new Stats();
-				container.appendChild( stats.dom );
+				// Ground plane and fog: examples for applying additional children and new property values to the loaded scene
 
-				//
+				var geometry = new THREE.PlaneBufferGeometry( 20000, 20000 );
+				var material = new THREE.MeshStandardMaterial( { roughness: 1 } );
```

Comment:
My reason for using MeshStandardMaterial in this place was that it is more intuitive (and the performance aspect should not play a role here): 

My goal was to avoid the strangely shaped highlight on the ground plane (seen when zooming out), and indeed that could be done by decreasing the default shininess in a MeshPhongMaterial as well, with the same visual effect.

But setting roughness from 0.5 to 1 in a MeshStandardMaterial is much more intuitive than setting shininess from its default value 30 to something like 0.01, isn't it?

This nice little aspect I wanted to demonstrate here en passant.